cocos/asset/assets/asset-enum.ts

```diff
@@ -22,10 +22,12 @@
  THE SOFTWARE.
 */
 
-import { Address, Filter as GFXFilter, Format } from '../../gfx';
+import { Address, Filter, Format } from '../../gfx';
 
 // define a specified number for the pixel format which gfx do not have a standard definition.
-const CUSTOM_PIXEL_FORMAT = 1024;
+enum CustomPixelFormat {
+    VALUE = 1024,
+}
```

Comment:
What's the benefit to change to enum?

Reply:
For inlining value defined in enums.
For example:

```
// old code
const CUSTOM_PIXEL_FORMAT = 1024;
enum PixelFormat {
    RGB_A_PVRTC_2BPPV1 = CUSTOM_PIXEL_FORMAT,
}
```
RGB_A_PVRTC_2BPPV1 will not be able to be inlined, and `inline enum` plugin will also complain that since it could not find where CUSTOM_PIXEL_FORMAT is defined.
Currently, `inline enum` rollup plugin only search all enum values, and will not search `const variables` so it doesn't know what `CUSTOM_PIXEL_FORMAT` is.

And if we define the private const value in another enum `CustomPixelFormat`, `inline enum` plugin will know that.

Reply:
Good to know. Does the plugin plan to support const variables?

Reply:
There is no plan for that, it will make the plugin more complex. There are too few usage like this in our engine code. So it's no in high priority. Keep the plugin code simpler is more important.
BTW, it plugin code is at https://github.com/cocos/cocos-ccbuild/tree/master/modules/build-engine/src/engine-js/rollup-plugins/inline-enum